pkg/core/interop/context.go

```diff
@@ -405,7 +405,15 @@ func (ic *Context) IsHardforkEnabled(hf config.Hardfork) bool {
 	if ok {
 		return ic.BlockHeight() >= height
 	}
-	return len(ic.Hardforks) == 0 // Enable each hard-fork by default.
+	if len(ic.Hardforks) == 0 {
+		return true // Enable each hard-fork by default if no hardforks specified in the config.
+	}
+	for _, cfgHf := range config.Hardforks {
```

Comment:
Why don't we solve this in `NewBlockchain()` once (preparing a complete proper hf map for internal use) and not do these dances here?

Reply:
Fixed.